eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -48,18 +48,17 @@ library
   build-depends:
                 bytestring,
                 cardano-binary,
-                cardano-crypto-class,
+                -- cardano-crypto-class,
```

Comment:
if these are not needed, we should delete them

Reply:
done